.gitignore

```diff
@@ -16,3 +16,5 @@ pa11y-results.json
 # Ignore VS code directories
 *.code-workspace
 .vscode/
+
+Gemfile.lock
```

Comment:
The lockfile is encoding ARM-specific dependencies that could be breaking some installations. If we ignore it, we may have more success installing across devices — at the expense of standardization, but maybe that's worth it